src/plugins/timeConductor/plugin.js

```diff
@@ -64,13 +64,15 @@ function validateConfiguration(config, openmct) {
             return m;
         }, {});
 
-    return config.menuOptions.map(function (menuOption) {
+    return config.menuOptions.map(menuOption => {
+        let message = '';
         if (menuOption.timeSystem && !systems[menuOption.timeSystem]) {
-            return `Time system '${menuOption.timeSystem}' has not been registered: \r\n ${JSON.stringify(menuOption)}`;
```

Comment:
Which rule is being violated here?

Reply:
array-callback-return. There was no need to change it to an arrow function, just to make a return a single value at the end of the function.

Reply:
I don't understand how it violates array-callback-return. I don't see anything in [the docs](https://eslint.org/docs/rules/array-callback-return) that states that it has to have a single return statement, in fact one of the examples includes multiple return statements.

Reply:
Don't worry about it, let's just get this merged. It's compatible with our code style.